modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java

```diff
@@ -1112,6 +1114,12 @@ private Map<String, String> _addObjectDefinitions(
 				continue;
 			}
 
+			if (objectDefinition.getAccountEntryRestricted()) {
+				objectDefinitionMap.put(
+					objectDefinition.getExternalReferenceCode(),
```

Comment:
Since we are  now fetching the data using local service to call the PATCH method, Couldn't it be better to use a map with name as key instead of ERC? So we don't need anymore to oblige a ERC in order to set an account restriction @joseabelenda